packaging/nuget/Particular.CodeRules.nuspec

```diff
@@ -2,8 +2,10 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
     <metadata>
         <id>Particular.CodeRules</id>
-        <version>$version$</version>
+        <version>$version$-beta</version>
```

Comment:
If you use GitVersion the version "calculated" by it would already contain beta, alpha... whatever supported by GitVersion